react/package.json

```diff
@@ -4,6 +4,7 @@
   "main": "index.js",
   "license": "MIT",
   "dependencies": {
+    "react-intl": "^2.4.0",
```

Comment:
This dependency was [already declared in a different version](https://github.com/vtex-apps/store-footer/pull/19/files#diff-d910ff25cec603033e405bb730ba3d47R11). Is this necessary?